fdbclient/ManagementAPI.actor.cpp

```diff
@@ -1062,12 +1063,30 @@ struct AutoQuorumChange final : IQuorumChange {
 	                                       Reference<IClusterConnectionRecord> ccr,
 	                                       int desiredCount,
 	                                       std::set<AddressExclusion>* excluded) {
+		ClusterConnectionString cs = ccr->getConnectionString();
+		if (oldCoordinators.size() != cs.hostnames.size() + cs.coords.size()) {
```

Comment:
This will cause `AutoQuorumChange::getDesired()` to select new coordinators. Is this desired?

Reply:
This means some of the coordinators hostnames are invalid/unreachable, we return false here as the current requirement is all coordinators are reachable: https://github.com/apple/foundationdb/blob/5e8c3130e6ba775fbd4de1a0c836414a9534ab25/fdbclient/ManagementAPI.actor.cpp#L1088.